### PR TITLE
test: fix for gh_6128 test to avoid flakiness

### PR DESCRIPTION
**Pull request by Lord-KA**
Followup to #6128

NO_DOC=test fix
NO_CHANGELOG=test fix

### CODE REVIEW

File: test/app-luatest/gh_6128_background_mode_test.lua [hunk 1]
```diff
@@ -60,7 +60,7 @@ g.after_test("test_background_mode_box_cfg", function(cg)
 end)
 
 g.test_background_mode_box_cfg = function(cg)
-    t.helpers.retrying({timeout = 2, delay = 0.01}, function()
+    t.helpers.retrying({timeout = 5, delay = 0.5}, function()
```
Comment (ligurio) [inline test/app-luatest/gh_6128_background_mode_test.lua:63]: Nit: I would add a comment where say that increased delay is required here.

**Review by ligurio — APPROVED** [review]
Gleb, thanks for fixing flakiness! I have no objections, so LGTM.
However, I've added a minor comment, see inline.

**Comment by Lord-KA** [comment]
@ligurio, thanks for the alternative fix! Your solution solves the underneath problem, so lets stick with it. I can't set a gh approve because this is my PR, but anyway LGTM.

**Comment by igormunkin** [comment]
Also backported to 2.11 and 2.10 in scope of 6381fc211 and 6a2c73f81 respectively.

**Pull request merged**